server/src/modules-resolver.ts

```diff
@@ -12,3 +14,17 @@ export function findPathToModule(dir: string, moduleName: string): string|undefi
         return undefined;
     }
 }
+
+export function findPathToSdk(dir: string, moduleName: string): string | undefined {
```

Comment:
Will re-create this issue to rename the function. My old one got outdated due to force-pushing.